bgpd/bgp_evpn_vty.c

```diff
@@ -2816,6 +2838,8 @@ static void evpn_show_route_rd_all_macip(struct vty *vty, struct bgp *bgp,
 		if (!dest)
 			continue;
 
+		bgp_dest_unlock_node(dest);
```

Comment:
just for completeness can we move all of these lookup-based unlocks to a point after we are finished accessing dest.
There are a bunch in bgp_route.c and bgp_updgrp_adv.c also.

Reply:
Sure, will do that.